### ISSUE TITLE
Flow property in WaterQualityObserved datamodel

### ISSUE THREAD

**Issue by vanmeule** [smart-data-models/dataModel.WaterQuality#2]
Within the [SCOREwater project](https://scorewater.eu/) we have applied the WaterQualityObserved datamodel to harmonize data collected by sensors from different vendors measuring waste water from construction sites in the City of Gothenburg. What was inconvenient was that there is no explicit property to include the flow (the amount of water flowing through the manhole/pipe in which the sensors have been installed). This flow is very relevant since it can explain changes in the concentration of chemical agents (for instance if it starts to rain the concentration of a pollutant coming from another source might drop). We are curious if others agree that flow might be useful property to add to the data model.

**Comment by albertoabellagarcia** [comment]
I understand the issue. It is also true that we have the flow in these data models:
-  [https://github.com/smart-data-models/dataModel.Environment/tree/master/WaterObserved](https://github.com/smart-data-models/dataModel.Environment/tree/master/WaterObserved), 
- [https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Pump](https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Pump)
- [https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Pipe](https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Pipe)
- [https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Valve](https://github.com/smart-data-models/dataModel.WaterDistributionManagementEPANET/tree/master/Valve)
- [https://github.com/smart-data-models/dataModel.WasteWater/tree/master/WasteWaterJunction](https://github.com/smart-data-models/dataModel.WasteWater/tree/master/WasteWaterJunction)

We are not against a new attribute for that as long as there is an actual use case, but could you review these data models just in case you were 'piling' all your data in the same payload and you should use one of these? 
Thanks

**Comment by vanmeule** [comment]
We are using the WaterQualityObserved data model for two use cases now: 
a) to record water quality going in and out of construction sites in Gothenburg in order to be able to asses whether the water quality deteriorates at the construction site. For that purpose, temperature, conductivity, total suspended solids and pH are measured in addition to flow;
b) water quality in the sewer system in Barcelona at three locations for predictive maintenance purposes. For this purpose, temperature, conductivity, total suspended solids, salinity and pH are measured in addition to flow, together with concentartions of various chemical agents. 

For both use cases, we looked at the WaterObserved data model as well. Applying this data model to be able to incorporate flow would entail publishing two entities for each measurement, a WaterQualityObserved one and a WaterObserved one. This makes using the data more difficult for our users since they have to come up with a procedure to link the two. We have tried this approach using two other data models (GreenSpaceRecord to record soil moisture and WeatherObserved to record the corresponding air temperature) and it makes it more difficult to use the data. From a rather self explanatory subscription for one entity type we have to explain how to access the air temperature for an incoming GreenSpaceRecord. In the soil moisture use case we think this is acceptable since measuring air temperature with soil moisture is rather unusual (most people are not very interested in these air temperature measurements, they just want the soil moisture measurements). But in case of WaterQualityObserved, flow is very important to be able to explain sudden changes in concentration.
